apps/src/javalab/JavalabView.jsx

```diff
@@ -204,11 +206,18 @@ const styles = {
   },
   buttons: {
     display: 'flex',
-    alignItems: 'center'
+    alignItems: 'center',
+    flexDirection: 'row-reverse',
```

Comment:
why use 'row-reverse' here instead of swapping the position of the buttons in `render`?